pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -465,6 +465,7 @@ public void setupInput() throws Exception {
         if (sourceSpec.getClassName().isEmpty()) {
             PulsarSourceConfig pulsarSourceConfig = new PulsarSourceConfig();
             pulsarSourceConfig.setTopicSerdeClassNameMap(sourceSpec.getTopicsToSerDeClassNameMap());
+            pulsarSourceConfig.setTopicsPattern(sourceSpec.getTopicsPattern());
```

Comment:
there seems to be some overlap with:
https://github.com/apache/incubator-pulsar/pull/1903

Reply:
@jerrypeng 
yes, I have rebased the change and now, we should not see any overlap here.